modules/exploits/linux/http/netgear_r7000_cgibin_exec.rb

```diff
@@ -40,7 +40,8 @@ def initialize(info = {})
           'Space'       => 1024,
           'DisableNops' => true,
           'BadChars'    => "\x20"
-        }
+        },
+      'DefaultOptions'  => { 'WfsDelay' => 10}
```

Comment:
I plan to test this when I get home later today, but I changed it based on your review

> This is what WfsDelay is for. You can safely remove this.

Should I also change the payload type to mettle?